docs/epics-and-tasks.md

```
@@ -0,0 +1,186 @@
+2️⃣ Повний список EPIC → TASK (High-Level Roadmap)
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)